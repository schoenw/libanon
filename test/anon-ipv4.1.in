1.2.3.4
1.2.3.3
0.0.0.0
1.1.1.1
255.255.255.254
255.255.255.255
2.6.8.15
2.6.3.83
1.2.3.3
1.2.3.4
6.3.11.123
112.28.3.2
220.201.45.30
1.2.3.4
