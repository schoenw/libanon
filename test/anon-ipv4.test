#!/bin/bash
#
# Shell script for regression testing libanon (anon-ipv4).
#
# $Id$
#

ANON=../src/anon
PASSPHRASE=testing

RC=0
for file in anon-ipv4.*.in; do
    $ANON ipv4 -p $PASSPHRASE $file \
	| diff -u `basename $file .in`.out -
    if [ $? -ne 0 ]; then
 	RC=1
    fi
done

exit ${RC}