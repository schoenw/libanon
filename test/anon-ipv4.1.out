236.157.211.10
236.157.211.12
237.113.211.225
236.159.20.242
0.0.127.6
0.0.127.7
238.6.10.23
238.6.7.179
236.157.211.12
236.157.211.10
233.3.231.131
136.126.15.210
61.17.221.254
236.157.211.10
